app/policies/entity_policy.rb

```diff
@@ -0,0 +1,31 @@
+class EntityPolicy < ApplicationPolicy
+  def create?
+    user.admin?
+  end
+
+  def destroy?
+    user.admin?
+  end
+
+  def see?
+    user.admin?
```

Comment:
Everybody should be able to see.  Return `true`.

Reply:
<img width="1148" alt="Screen Shot 2021-10-05 at 1 45 11 PM" src="https://user-images.githubusercontent.com/10540193/135967125-fb3204b1-22f1-4909-9163-fe7768ce3438.png">
<img width="1165" alt="Screen Shot 2021-10-05 at 1 45 01 PM" src="https://user-images.githubusercontent.com/10540193/135967218-fc2a6054-9030-4e46-981a-d3d1ced5ac64.png">

Wala na po kasing natitirang action na pwedeng gawin si ordinary user(Don) kaya nilagyan ko niyan para hindi niya makita `Actions`